### DISCUSSION TITLE
coco bbox and area changed when read into sahi

### DISCUSSION THREAD

**Discussion by nye0** [obss/sahi#681] [Q&A] [answered]
The COCO annotation file was attached.
```
# python
from sahi.utils.coco import Coco
from sahi.utils.file import load_json
coco_dict = load_json(COCO_annotation_PATH)
coco = Coco.from_coco_dict_or_path(coco_dict)
```
when i check sample 428:
```
coco.images[428].annotations
# bbox: [131, 130, 1,1],
# area: 0
```
but when i check the coco_annotation file, by search the string "131, 130, " and found:

```
# bbox: [131, 130, 2, 2]
# area: 4
```
and then i check all the bbox, found that the "bbox" width and height were all shrink by 1, and the "area" also become smaller(randomly).

[coco.zip](https://github.com/obss/sahi/files/9690259/coco.zip)

**Comment by devrimcavusoglu** [comment, marked as answer]
Hi @nye0. We're not able to recreate your problem with the latest sahi version `0.10.7`, can you try it on colab or sth and share with us ?
